packages/mdc-toolbar/README.md

```diff
@@ -141,6 +141,14 @@ Icons can be added as anchor tags, `span`s, or `button`s to `mdc-toolbar`. There
 `mdc-toolbar__menu-icon` represents the left most icon in `mdc-toolbar` usually to the left of `mdc-toolbar__title`.
 `mdc-toolbar__icon` represents any icons placed on the right side of an `mdc-toolbar`.
 
+When using `svg` icons, ensure you wrap the `svg` element in a `span` and include the `mdc-toolbar__icon` class.
```

Comment:
SVG's don't have a `::before` or `::after` element, which we need for the ripple, so I added this section to the so developers are aware.